docs/nidcpower/examples.rst

```diff
@@ -1,15 +1,6 @@
 Examples
 ========
 
-nidcpower_source_delay_measure.py
```

Comment:
I've noticed that on different systems, this generates in different order. I think this should be deterministic.
Opened #403.

Reply:
Reverted.